test/e2e/support/utils/mock-or-transform.js

```diff
@@ -0,0 +1,34 @@
+import fetch from 'node-fetch';
```

Comment:
We add those comments for dependencies in all files. It should look as follows:

```js
/**
 * External dependencies
 */
import fetch from 'node-fetch';

/**
 * Internal dependencies
 */
import { getJSONResponse } from './get-json-response';
```

We use 3 groups in the following order:
- external
- WordPress
- internal

Reply:
Ok, updated all files accordingly.